tests/installation/configure_dasd.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Activates multipath when multipath activation message appears
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use parent 'y2_installbase';
+use testapi;
+use strict;
+use warnings;
+
+sub run {
+    my $dasd_channel = "0.0.0150";
+
+    my $disk_activation_controller = $testapi::distri->get_disk_activation();
```

Comment:
we omit `_controller` as it is obvious that all that comes from `$testapi::distri` are controllers, and even if some day we decide to merge controllers and pages, we don't need to change tests.

Reply:
Makes sense, I ll rename it!